src/deployment/windows.md

```diff
@@ -108,6 +108,30 @@ so, Codemagic requires
 [associating the Azure Active Directory
 and Partner Center accounts][azureadassociation].
 
+#### Github Actions CI/CD
+
+Github actions may use the
+[Microsoft Dev Store CLI](https://learn.microsoft.com/en-us/windows/apps/publish/msstore-dev-cli/overview)
```

Comment:
The learn.microsoft.com website redirects to localized content only if the locale segment is removed from the path:

```suggestion
[Microsoft Dev Store CLI](https://learn.microsoft.com/windows/apps/publish/msstore-dev-cli/overview)
```